### DISCUSSION TITLE
长假长假！

### DISCUSSION THREAD

**Discussion by Jason555647** [share121/inter-knot#5473] [其他]
好消息：明天下午就可以放假了
坏消息：我这边只放4天
（求安慰）🤧

**Comment by BJ-cmyk** [comment]
我这里放三天😡😡😡

**Reply by Jason555647** [reply]
天啊，给你一个拥抱

**Comment by shancun42** [comment]
我们是七天，还好还好

**Comment by anbi-demala** [comment]
我们也是七天<img src="https://bbs-static.miyoushe.com/static/2024/05/10/5eb4431537387b22233ac3c6da58189f_4285892181442425038.png" alt="知更鸟 唱歌" width="75" >

**Comment by bigetree** [comment]
我两天

**Reply by bigetree** [reply]
【墨镜】
